win32ss/user/ntuser/desktop.c

```diff
@@ -1503,7 +1503,7 @@ IntPaintDesktop(HDC hDC)
                 PWCHAR pstr = wszzVersion;
                 for (i = 0; (i < ARRAYSIZE(VerStrs)) && *pstr; ++i)
                 {
-                    VerStrs[i].n = wcslen(pstr);
+                    VerStrs[i].n = (ULONG)wcslen(pstr);
```

Comment:
Not a fan of this one

Reply:
You don't have to be a fan, it only needs to be correct, which it is.

Reply:
wcslen in win32k. 😓

Reply:
What else would you want to use?

Reply:
I added an inline wrapper lstrlenW that allows to get rid of a few hacks to wine code and used it here, too.